src/endpoint/s3/ops/s3_get_bucket_policy_status.js

```diff
@@ -0,0 +1,23 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const S3Error = require('../s3_errors').S3Error;
+
+/**
+ * https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetBucketPolicyStatus.html
+ */
+async function get_bucket_policy_status(req) {
+    const reply = await req.object_sdk.get_bucket_policy_status({ name: req.params.bucket });
```

Comment:
If we are using the same commands as get_bucket_policy. Maybe just move _is_policy_public to here and just call req.object_sdk.get_bucket_policy. WDYT?
```suggestion
    const reply = await req.object_sdk.get_bucket_policy({ name: req.params.bucket });
    if (!reply.policy) throw new S3Error(S3Error.NoSuchBucketPolicy);
    return _is_policy_public(reply.policy);
```

Reply:
seems much simpler, I'll change that